base-notebook/start.sh

```diff
@@ -16,6 +16,9 @@ if [ $UID == 0 ] ; then
                 chown -R $NB_UID "$d"
             fi
         done
+    elif [[ -d /home/$NB_USER/work && $(stat -c %U /home/$NB_USER/work) != "$NB_USER" ]]; then
```

Comment:
I'll test it when we have our kube ready. For the moment, on our Marathon install, it does not seem possible to set the user id